cheat.php

```diff
@@ -713,6 +776,46 @@ function ExecuteRequest( $Method, $URL, $Data = [] )
 	return $Data;
 }
 
+function GetLocalScriptHash( )
+{
+	return sha1_file( __FILE__ );
```

Comment:
Actually since you call this only once, a separate function is not necessary. Just move this `sha1_file` where you assign it.

And so I don't make a new comment, you forgot to remove `GetCurlRepository` function.